### PR TITLE
Update popularity values for collaborations

### LINKED ISSUE #787: [Idea]: Go through collaborations and update popularity (followers)
### Describe your idea / enhancement

Go through the collaborations and update follower numbers. Most of them are very similar still, but some made big jumps. https://github.com/alveusgg/alveusgg/blob/main/apps/website/src/data/collaborations.ts#L67

### How would this idea / enhancement be beneficial?

Better highlight collaborations of (now) popular content creators.

### PR DESCRIPTION
**Pull request by shradiphylleia**
## Describe your changes

Updated the follower/popularity count 

Resolves #787

## Notes for testing your change

The updated follower count can be seen on the collaboration page

### CODE REVIEW

File: apps/website/src/data/collaborations.ts [hunk 1]
```diff
@@ -428,12 +428,12 @@ const collaborations: Collaboration[] = (
         {
           name: "PeachJars",
           image: peachJars,
-          popularity: 720_000, // Twitter followers, 2024-04-07
+          popularity: 744_107, // Twitter followers, 2024-10-4
```
Comment (MattIPv4) [inline apps/website/src/data/collaborations.ts:431]:
```suggestion
          popularity: 744_00, // Twitter followers, 2024-10-4
```
Rounded

File: apps/website/src/data/collaborations.ts [hunk 2]
```diff
@@ -224,12 +224,12 @@ const collaborations: Collaboration[] = (
...
           name: "Eret",
           image: eret,
-          popularity: 1_200_000, // Twitch followers, 2024-04-11
+          popularity: 1_100_000, // Twitch followers, 2024-10-4
         },
         {
           name: "Ava Kris Tyson",
           image: avaKrisTyson,
-          popularity: 435_000, // Twitter followers, 2024-04-11
+          popularity: 426_552, // Twitter followers, 2024-10-4
```
Comment (MattIPv4) [inline apps/website/src/data/collaborations.ts:232]:
```suggestion
          popularity: 426_500, // Twitter followers, 2024-10-4
```
Rounded

**Review by MattIPv4 — CHANGES_REQUESTED** [review]

**Comment by shradiphylleia** [comment]
@MattIPv4 thanks for the review i do think the suggestions must have been added. Please let me know otherwise. thanks

File: apps/website/src/data/collaborations.ts [hunk 3]
```diff
@@ -529,7 +529,7 @@ const collaborations: Collaboration[] = (
         {
           name: "YungJeff",
           image: yungJeff,
-          popularity: 40_000, // Twitter followers, 2024-04-07
+          popularity: 39_372, // Twitter followers, 2024-10-4
```
[the remark this answers is not in the document]
Comment (shradiphylleia) [reply apps/website/src/data/collaborations.ts]: sorry must have missed it. have committed the suggested changes now. thanks for the review

File: apps/website/src/data/collaborations.ts (hunk 1 above)
Comment (shradiphylleia) [reply apps/website/src/data/collaborations.ts]: @MattIPv4 noticed it now. The rounded-up value must be 744_000 or did you mean to go by that number?
Let me know and I will append the zero.

Comment (MattIPv4) [reply apps/website/src/data/collaborations.ts]: I definitely missed a zero, good spot

Comment (shradiphylleia) [reply apps/website/src/data/collaborations.ts]: updated the same. please have a look and let me know if any other changes could be of use

**Review by MattIPv4 — APPROVED** [review]

**Comment by MattIPv4** [comment]
Thanks for your work on this 💚

**Pull request merged**